src/attribute_type.rs

```diff
@@ -1243,6 +1246,7 @@ impl FromStr for SdpAttribute {
             "mid" => Ok(SdpAttribute::Mid(string_or_empty(val)?)),
             "msid-semantic" => parse_msid_semantic(val),
             "ptime" => Ok(SdpAttribute::Ptime(val.parse()?)),
+            "ice-pacing" => Ok(SdpAttribute::IcePacing(val.parse()?)),
```

Comment:
This needs to ensure that the value has the proper range which is 0-9999999999, 10 digit base 10 integer.

Reply:
I would recommend adding a specific error for this (instead of a generic with a string).